doc/source/admin/dependency_resolvers.rst

```diff
@@ -0,0 +1,143 @@
+Dependency Resolvers in Galaxy
+==============================
+
+There are two parts to building a link between Galaxy and command line bioinformatics tools: the tool XML that
+specifies a mapping between the Galaxy web user interface and the tool command line and tool dependencies that specify
+how to source the actual packages that implement the tool’s commands. The final script that Galaxy submits to run a
+the job uses includes commands, such as changes to the ``PATH`` environment variable, that are generated by *dependency
+resolvers*. There is a default dependency resolver configuration but administrators can provide their own configuration
+using the ``dependency_resolvers_conf.xml`` configuration file in the Galaxy ``config/`` directory.
+
+The binding between tool XML and the tools they need to run is specified in the tool XML using *requirement*
+tags, for example
+
+.. code-block:: xml
+
+    <requirement type="package" version="0.7.10.039ea20639">bwa</requirement>
+
+In some cases these requirement tags can be specified without a version
+
+.. code-block:: xml
+
+    <requirement type="package">bedtools</requirement>
+
+The requirement turn into inputs to the dependency resolver. Each dependency resolver is thus given given one or
+two inputs: the name of the dependency to resolve and, in most cases, the version number of the
+dependency. By default, the dependency resolver configuration contains three items. First, the
```

Comment:
I'd move the part starting with "By default" to the `Default Dependency Resolvers` subsection.